api/resources.py

```diff
@@ -1,12 +1,18 @@
 from .schemas import *
 from .models import Category, Bot, User
+from app import pagination
 from flask_restful import Resource
 from app import db
 from flask import request, jsonify
 from werkzeug.security import generate_password_hash, check_password_hash
 from flask_jwt_extended import create_access_token, jwt_required
 
 
+pagination_schema = lambda current_page, page_obj: {
```

Comment:
if you use the function in multiple places and it has a name, it doesn't make sense to use lambda